website/docs/sdk-reference/react.mdx

```diff
@@ -977,6 +987,47 @@ If you do not want to expose the SDK key or the content of the config JSON file,
 
 Also, we recommend using [confidential targeting comparators](../targeting/targeting-rule/user-condition.mdx#confidential-text-comparators) in the Targeting Rules of those feature flags that are used in the frontend/mobile SDKs.
 
+## Using multiple providers
+
+This section explains how can you use multiple providers in one application with ReactSDK.
+
+It can be real life scenario when you arrange your feature flags into multiple configs. These configs have different sdkKeys and in your code you have to differentiate those with ConfigCatProvider's attribute called `id`.
```

Comment:
To access a feature flag's value, your code needs to refer to that Config where that particular feature flag is located. This is what a `ConfigCatProvider` does for you; it refers to the Config by specifying the Config's `sdkKey`.

If your code needs access to feature flags from multiple Configs, you will need multiple `ConfigCatProvider`s - each pointing to a Config by specifying that particular Config's `sdkKey`.

And when your code wants to access a particular feature flag, it should refer to the specific `ConfigCatProvider` that is associated with the Config where that particular feature flag is located. This is exactly what the `providerId` allows your code to do: to refer to the specific `ConfigCatProvider` that is associated with the feature flag it wants to access.